types/order.ts

```diff
@@ -27,4 +33,9 @@ export interface Order {
     shipping_cost_ex_tax: string;
     total_inc_tax: string;
     total_tax: string;
-}
\ No newline at end of file
+}
+
+export interface ShippingOrder {
```

Comment:
Let's extend this from `Order` since the and rename it to be more explicit. Here's my recommendation:
```ts
export interface ShippingAndProductsOrder extends Order {
  //...
}
```